packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -43,6 +47,12 @@ export default class HostItem extends Component {
     return host && host.power_state === 'Running'
   }
 
+  _isMaintained = createSelector(
+    () => this.props.HVSupportedVersions,
```

Comment:
You might need to check this but I think every time the supported versions will be fetched, a new object will be created from the JSON, which will trigger a recompute from the selector since it's a new reference.

Reply:
With the subscription, for each time we execute the cb we are doing a [deep comparison](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/xo/index.js#L219) between the response and the cache, and only in case that the `deep comparison` return false, we update the cache.